docs/developers/coredev/release.md

```diff
@@ -2,50 +2,61 @@
 
 # Release guide
 
-This guide documents `napari`'s release process. Most required tools mentioned here are in https://github.com/napari/napari-release-tools
+This guide documents `napari`'s release process as of 0.6.0.
+Most required tools mentioned here are in https://github.com/napari/napari-release-tools.
 
-# Timeline
+## Timeline
 
-Currently, we are releasing a new napari version every 2-3 weeks. Release candidates are made available as a reasonableness check for 1-24h before the full release is published: since the release cadence is so high, any new bugs can be quickly rectified, and more quickly identified by making a full release.
+Currently, we are releasing a major version every six to twelve months.
+We plan minor napari version releases every 3 months.
```

Comment:
What is considered major/minor? 0.6.0 is pretty major...

Reply:
Given how we have been doing releases, 0.6.0 would be a major version though it doesn't strictly follow sematic versioning. 0.6.1 would be a minor version.

Reply:
I don't think every 3 months is accurate for minor versions.

Reply:
```suggestion
Currently, we are releasing a major version every six to twelve months.
We plan minor napari version releases every 1-2 months.
```